src/app/three-examples/objects/loaders/obj-loader.directive.ts

```diff
@@ -3,9 +3,9 @@ import * as THREE from 'three';
 import { AbstractObject3D } from '../abstract-object-3d';
 import { AbstractModelLoader } from './abstract-model-loader';
 
-import '../../js/EnableThreeExamples';
-import 'three/examples/js/loaders/OBJLoader';
-import 'three/examples/js/loaders/MTLLoader';
+import * as THREE_OBJ from 'three/examples/jsm/loaders/OBJLoader';
```

Comment:
Couldn't we just do `import {OBJLoader} from 'three/...` and `import {MTLLoader} from 'three/...` one line below? That would seem cleaner to me.